release-notes-70.60.2.md

```diff
@@ -1,8 +1,12 @@
-!not-ready-for-release!
-
 #### Version Number
 ${version-number}
 
 #### New Features
+ - None
+
+#### Patch Fixes Included
+ - Added missing gdbm module
+    This module is required for Celery state merging.
```

Comment:
@isharov I don't really understand what I just wrote here.  Could you update with a better description?

Reply:
Done